src/sentry/static/sentry/app/views/settings/organizationMembers/inviteMember/roleSelect.jsx

```diff
@@ -31,7 +31,7 @@ class RoleSelect extends React.Component {
     const {disabled, enforceAllowed, roleList, selectedRole} = this.props;
 
     return (
-      <Panel className="new-invite-team">
+      <Panel className="new-invite-role">
```

Comment:
Do we need this class here? There don't seem to be any styles associated with it

Reply:
It isn't used. I corrected it as it was wrong, but I'm happy to take it out entirely.

Reply:
Fixed now.